content/about/_index.md

```diff
@@ -5,47 +5,51 @@ layout: about
 collaborators:
 - name: Amazon
   group: collaborator
-- name: CDC
+- name: Centers for Disease Control and Prevention
   group: collaborator
 - name: Change Healthcare
   group: collaborator
 - name: Defense Threat Reduction Agency
   group: collaborator
 - name: Facebook
   group: collaborator
-- name: Uptake
+- name: Google
```

Comment:
I can remember that there was a ?contracting? reason why it was stated like `With support from Google.org` but I cannot remember the deatils

Reply:
I can figure it out later by following up with them but for now it makes no sense to differentiate, as I explained in my above messages.